### DISCUSSION TITLE
Will this library properly function while scaling via multiple instances?

### DISCUSSION THREAD

**Discussion by devlzcode** [taskforcesh/bullmq#501] [Q&A]
I'd like to be able to process jobs across multiple instances and having zero overlap (aka the same job being processed at the same time, creating redundancy). For further context, I'm using this library for a jackpot-like game where each "join" request is processed one at a time (ideally multiple) and after a certain amount of time or amount of players, it will "end" automatically. In other words each server should be able to process a "join" job while only the first server who received it should be able to process the "end" job.

**Comment by manast** [comment]
A job in BullMQ is processed only once independently on the amount of workers you have. If you want to process the same job as many times as you have workers you can achieve it with redis streams: https://redis.io/topics/streams-intro
